.gitignore

```diff
@@ -12,6 +12,12 @@ bin/
 # All __pycache__
 __pycache__/
 
+# All test output folders
+tmp/
+
+# Logs
+log.txt
```

Comment:
So if I recall correctly, these log.txt/tmp folder show up not necessarily in the repo root directory but could be in one or more of the subdirectory. Could you revise it accordingly? 

Secondly, could you investigate the root cause and see if it is possible to prevent this behavior in the first place? i.e. generate these files in the unit test temporary folder if the above file/folder were generated during unit testing scenarios.

Reply:
I just tested it if you put log.txt in .gitignore it ignores this file doesn't matter which subdirectory it is in

Reply:
TIL. Good call. I need to brush up on my gitignore syntax. :)